src/helpers/Haversine.js

```diff
@@ -0,0 +1,18 @@
+export const getDistanceFromLatLonInKm = (lat1, lon1, lat2, lon2) => {
+  function deg2rad(deg) {
+    return deg * (Math.PI / 180);
+  }
+
+  var R = 6371; // Radius of the earth in km
```

Comment:
What I remember from when I wrote my own implementation almost 20 years ago 😅 is that for this to be accurate for people at different latitudes, you’d need to adjust this number.

Reply:
If we keep this code, we should update it to ES6 style. Thanks!